test/services/conversion/TransitAccompanyingDocumentConversionServiceSpec.scala

```diff
@@ -46,15 +50,59 @@ class TransitAccompanyingDocumentConversionServiceSpec
     with ScalaFutures
     with IntegrationPatience {
 
-  private val countries = Seq(Country("valid", "AA", "Country A"), Country("valid", "BB", "Country B"))
+  private val countries                 = Seq(Country("valid", "AA", "Country A"), Country("valid", "BB", "Country B"))
+  private val kindsOfPackage            = Seq(KindOfPackage("P1", "Package 1"), KindOfPackage("P2", "Package 2"))
+  private val documentTypes             = Seq(DocumentType("T1", "Document 1", transportDocument = true), DocumentType("T2", "Document 2", transportDocument = false))
+  private val additionalInfo            = Seq(AdditionalInformation("I1", "Info 1"), AdditionalInformation("I2", "info 2"))
+  private val sensitiveGoodsInformation = Nil
 
   implicit private val hc: HeaderCarrier = HeaderCarrier()
 
   val validModel = models.TransitAccompanyingDocument(
-    localReferenceNumber = "lrn",
+    localReferenceNumber = "MRN GOES HERE",
```

Comment:
Should this be LRN?

Reply:
Its actually the LRN that's pulled out into the TransitAccompanyingDocument model. I've added an additional mrn placeholder argument that we'll use when we have it (this is pulled from IE28 message but we haven't got that in yet)